Sources/BraintreePayPalMessaging/BTPayPalMessagingTextAlignment.swift

```diff
@@ -14,7 +14,7 @@ public enum BTPayPalMessagingTextAlignment {
     /// Text aligned to the right
     case right
 
-    var textAlignmentRawValue: PayPalMessageTextAlignment {
+    var textAlignmentRawValue: PayPalMessageTextAlign {
```

Comment:
Not sure if Android renamed the top level enum but if they did we should do the same

Reply:
Or if we feel that `alignment` is a better name we should stay with that across the board

Reply:
Confirmed!! We have the same on Android, I think alignment is ok